### PR TITLE
Temp plotting fix

### PR DESCRIPTION
**Pull request by tcromartie**
For be/rx combos without markers or colors in the plotting dict, assigns a black 'x' and throws a warning to avoid crashing.

### CODE REVIEW

**Review by swiggumj — CHANGES_REQUESTED** [review]
I think something may have gone wrong with pulling latest updates since lots of functions are being removed as a part of this PR. If that's also getting fixed and it's not too much trouble, it might also be good to fix bare exceptions.

**Comment by tcromartie** [comment]
Oh it's a conflict with my plotting PR, I made a mistake.

File: src/timing_analysis/plot_utils.py [hunk 1]
```diff
@@ -2084,17 +2091,25 @@ def plot_residuals_orb(fitter, restype = 'postfit', plotsig = False, avg = False
             r_b_label = ""
         else:
             r_b_label = rcvr_bcknds[inds][0]
-        # Get plot preferences
-        if 'fmt' in kwargs.keys():
-            mkr = kwargs['fmt']
-        else:
+    # Get plot preferences
+    if 'fmt' in kwargs.keys():
```
Comment (JPGlaser) [inline src/timing_analysis/plot_utils.py:2095]: It looks like possible indentation changes between the old file and the new. Is that intended?

~ Joe G.

**Review by swiggumj — APPROVED** [review]
My previous concerns have been addressed, but it would be good to resolve a couple comments from Joe G. before merging.

**Pull request merged**